pokemongo_bot/cell_workers/pokemon_transfer_worker.py

```diff
@@ -24,21 +24,31 @@ def work(self):
                 keep_best, keep_best_cp, keep_best_iv = self._validate_keep_best_config(pokemon_name)
 
                 if keep_best:
-                    order_criteria = 'cp'
-                    limit = keep_best_cp
+                    best_pokemon_ids = set()
+                    order_criteria = 'none'
+                    if keep_best_cp >= 1:
+                        cp_limit = keep_best_cp
+                        best_cp_pokemons = sorted(group, key=lambda x: x['cp'], reverse=True)[:cp_limit]
```

Comment:
Can you sort by cp then iv (x['cp'], x['iv'])? if you have pokemon with the same amount of CP #1464